index.html

```diff
@@ -206,36 +206,41 @@ <h3>
         &lt;button&gt;Verify Identity&lt;/button&gt;
         &lt;script&gt;
           const button = document.querySelector("button");
-          button.addEventListener("click", async () =&gt; {
-            try {
-              const credential = await navigator.credentials.get({
-                digital: {
-                  requests: [{
-                    protocol: "example-protocol",
-                    data: { /* request data */ }
-                  }]
+          button.addEventListener("click", async () => {
+            // Check for DC API and protocol support
+            if (window.DigitalCredential && DigitalCredential.userAgentAllowsProtocol('example-protocol')) {
```

Comment:
We probably just want:

```suggestion
            if (DigitalCredential?.userAgentAllowsProtocol("example-protocol")) {
```